static/js/compose.js

```diff
@@ -282,6 +282,25 @@ export function create_message_object() {
     return message;
 }
 
+export function send_test_notification_message() {
+    const notification_bot = "notification-bot@zulip.com";
+    const notification_bot_id = page_params.zulip_user_bots[notification_bot];
+    const message_object = {
+        type: "private",
+        content: i18n.t("__emoji__ Hey, notification bot send me a message!", {
+            emoji: ":smile:",
+        }),
+        private_message_recipient: notification_bot,
+        sender_id: page_params.user_id,
+        queue_id: page_params.queue_id,
+        reply_to: notification_bot,
+        to: [notification_bot_id],
+        to_user_ids: notification_bot_id,
+    };
+
+    send_message(message_object);
+}
```

Comment:
@gnprice @alya thoughts on this UX?  I could imagine us doing a separate API endpoint rather than having there be a message we sent.

The other API we could use would be to effectively call the slash command backend, which would result in there not being a message from the user to notification-bot (which feels cleaner to me).

Reply:
Agreed that it would be cleaner. I think it's a confusing experience for the user to have a message from them that they didn't actually send. We could tag it as an automated message (e.g. using brackets at the beginning), but much better to avoid the confusion in the first place than to try to explain it.